extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-sqlite/src/main/java/org/apache/guacamole/auth/sqlite/SQLiteGuacamoleProperties.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.guacamole.auth.sqlite;
+
+import org.apache.guacamole.properties.BooleanGuacamoleProperty;
+import org.apache.guacamole.properties.IntegerGuacamoleProperty;
+import org.apache.guacamole.properties.StringGuacamoleProperty;
+
+/**
+ * Properties used by the SQLite Authentication plugin.
+ */
+public class SQLiteGuacamoleProperties {
+
+    /**
+     * This class should not be instantiated.
+     */
+    private SQLiteGuacamoleProperties() {}
+
+    /**
+     * The absolute path of the SQLite database containing the Guacamole
+     * authentication tables.
+     */
+    public static final StringGuacamoleProperty SQLITE_DATABASE =
+            new StringGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "sqlite-database"; }
+
+    };
+
+    /**
+     * Whether a user account within the database is required for authentication
+     * to succeed, even if the user has been authenticated via another
+     * authentication provider.
+     */
+    public static final BooleanGuacamoleProperty
+            SQLITE_USER_REQUIRED = new BooleanGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "sqlite-user-required"; }
+
+    };
+
+    /**
+     * The maximum number of concurrent connections to allow overall. Zero
+     * denotes unlimited.
+     */
+    public static final IntegerGuacamoleProperty
+            SQLITE_ABSOLUTE_MAX_CONNECTIONS =
+            new IntegerGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "sqlite-absolute-max-connections"; }
+
```

Comment:
Does these actually apply to SQLite??

Reply:
Which?

Reply:
All of the connection-limiting properties.  Just not sure those actually apply to something file-based like SQLite?

Reply:
They're applicable. The connection-limiting properties deal with the default behavior of Guacamole connections, not database connections.

The hostname and port properties definitely wouldn't apply, though.

Reply:
Sounds good.  I think I got the rest of them removed, just wasn't sure about those.